hazelcast/src/main/java/com/hazelcast/instance/BuildInfo.java

```diff
@@ -35,14 +35,21 @@
     private final boolean enterprise;
     private final byte serializationVersion;
 
+    private final String jetVersion;
+    private final String jetBuild;
+    private final String jetRevision;
```

Comment:
adding these fields to `toString` method can be helpful for diagnostic purposes.